client/state/data-layer/wpcom/experiments/index.js

```diff
@@ -31,9 +31,9 @@ const transformApiRequest = ( data ) => ( {
 export const handleFetchExperiments = ( action ) =>
 	http(
 		{
-			apiNamespace: 'wpcom',
+			apiNamespace: 'wpcom/v2',
```

Comment:
Use proper namespace, as mentioned in https://github.com/Automattic/wpcom-xhr-request/pull/33#issuecomment-739030816.